index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@700&family=Poppins:ital,wght@0,400;0,700;1,400;1,700&family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='css/main.css'>
+</head>
+
+<body>
+    <nav class="row nav-bar">
+        <div class="cell title">Juan Luis Palacios Perez</div>
+        <div id="menu-btn" class="cell">
+            <img src="imgs/Icon%20-%20Menu.svg" alt="menu" />
```

Comment:
- [ ] The `img` should be inside `nav` tag.
 Requirements ;
- The hamburger button is on the right and has a `nav` tag

Reply:
> * [ ]  The `img` should be inside `nav` tag.
>       Requirements ;
> 
>     * The hamburger button is on the right and has a `nav` tag

Concerning this comment, I see that you applied the necessary changes, but I don't know why this comment is still in this review. But you can forget about it now because you already solved this issue! 😎